src/style.css

```diff
@@ -0,0 +1,128 @@
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir Book", "Avenir Heavy", Arial, sans-serif;
```

Comment:
but you havent "Avenir Book" you have just "Avenir"
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96265844/263da501-f68b-42f9-a5c7-1dce4f006086)